saleor/graphql/product/mutations/products.py

```diff
@@ -540,6 +541,9 @@ class ProductInput(graphene.InputObjectType):
             "Note: this field is only used if a product doesn't use variants."
         )
     )
+    visible_in_listings = graphene.Boolean(
+        description="Determines if product is visible in product listings."
```

Comment:
"Determines if product is visible in product listings (doesn't apply to product collections)" - I've added info about collections since it's not obvious and can be useful to mention here.